docs/installation.md

```diff
@@ -6,13 +6,13 @@ This document can be used as a quick start guide to setup the development enviro
 
 | Component | Recommended |
 |------- |------|
-| Operating System | Ubuntu 18.04 / Windows 10 |
-| Docker Engine | 18.09 |
-| Docker-compose | 1.21.2 |
-| maven | 3.5.4 |
+| Operating System | Ubuntu\* 18.04 / Windows\* 10 |
+| Docker* Engine | 18.09 |
+| Docker* Compose | 1.21.2 |
+| Maven* | 3.5.4 |
 
```

Comment:
If the name of the tool is "docker-compose", then I think it should be shown that way.  This is not a case of using Docker without attribution.

Reply:
Compose is a Docker tool, the command is docker-compose. 
https://docs.docker.com/compose/